### DISCUSSION TITLE
An attempt to bring back annie/dark via custom CSS

### DISCUSSION THREAD

**Discussion by Ixve** [GooseMod/GooseMod#79] [Releases]
After seeing that Discord decided to kill off annie/dark, I went ahead and brought it back for the most part based on screenshots, there is some parts that I am missing the colors of, such as the little 'Unread Messages' label, friends list and search options, but other than that, it's essentially back.

Installation is easy if you have Topaz enabled, simply add it via the settings or open the Topaz terminal and paste the URL.

`https://www.github.com/c-um/anniedark/`

**Comment by CanadaHonk** [comment]
You're aware annie/dark was half stolen from an existing BD theme?

**Reply by Ixve** [reply]
> You're aware annie/dark was half stolen from an existing BD theme?

Not really, I havent been using themes that much, lately just stuck to the normal dark mode Discord offers, someone sent me the annie/dark override and I liked it more than the default dark mode so I stuck to it until they decided to just kill it off. Been using mods mostly for the plugins.

**Reply by CanadaHonk** [reply]
I made an issue in your repo
